frontend/src/app/[locale]/process/ProcessNext.tsx

```diff
@@ -14,6 +14,20 @@ import {
 
 import { USWDSIcon } from "src/components/USWDSIcon";
 
+export const gitHubIssueLink = (issueNumber: number) =>
+  Object.assign(
+    (chunks: ReactNode) => (
+      <Link
+        target="_blank"
+        className="usa-link--external text-bold"
+        href={`${ExternalRoutes.GITHUB_REPO}/issues/${issueNumber}`}
+      >
+        {chunks}
+      </Link>
+    ),
+    { displayName: "gitHubIssueLink" },
```

Comment:
The component needs a display name from an anon function, and the linter wouldn't accept adding the display name after creating it.

Reply:
I'm fine with this, but also found that my linter doesn't complain about this, but that could just be my local linter being weird...

```suggestion
const GithubIssueLink = ({
  issueNumber,
  chunks,
}: {
  issueNumber: number;
  chunks: ReactNode;
}) => (
  <Link
    target="_blank"
    className="usa-link--external text-bold"
    href={`${ExternalRoutes.GITHUB_REPO}/issues/${issueNumber}`}
  >
    {chunks}
  </Link>
);

const gitHubLinkForIssue = (issueNumber: number) => {
  const PartialIssueLink = (chunks: ReactNode) => (
    <GithubIssueLink issueNumber={issueNumber} chunks={chunks} />
  );
  PartialIssueLink.displayName = "GithubIssueLink";
  return PartialIssueLink;
};
```